src/zenml/cli/utils.py

```diff
@@ -146,6 +146,7 @@ def format_component_list(
             "component_name" if k == "name" else k: v
             for k, v in c.dict(exclude={"_superfluous_options"}).items()
         }
+        component_dict["active"] = "*" if key == active_component else ""
```

Comment:
I would make this the first column rather than the last. Find attached example here from kubectl. 

![image](https://user-images.githubusercontent.com/4792082/146527911-ac9467ab-53cb-4ad9-bdf8-e7e8948cf7ec.png)

I am also a fine of capitalizing the columns. Maybe we can add that in too? :-)

Reply:
Fixed the order which made my `orchestrator` issue moot. (Learnt something about dictionary order preservation in Python along the way!)

Also capitalised the column headers.